### ISSUE TITLE
keyboard focus outline is not visible on cards in Learn page

### ISSUE THREAD

**Issue by jonboiser** [learningequality/kolibri#7651]
### Observed behavior

In this video [from this link](https://kolibri-beta.learningequality.org/en/learn/#/classes), I shift-tab until I am focused on the skip-link, hit enter, hit tab once, then am focused on the first card "Jessica's class" (I can confirm this because I keypress "Enter" and am taken to the right page). Yet, there is no green outline on the card. 

(Notice that some elements on the top nav aren't outlined either while back tabbing)

![CleanShot 2020-10-29 at 11 42 56](https://user-images.githubusercontent.com/10248067/97618224-027a4880-19dc-11eb-8393-28e7e12d4209.gif)

(This happens in a lot of other areas, but fixing this in Learn will probably fix it globally)

### Expected behavior

Cards (which are anchor tags under the hood) get a green focus outline while in the keyboard modality.

### User-facing consequences

It's hard for keyboard-only users to know where they are focusing.

### Errors and logs

…

### Steps to reproduce

…

### Context

…
